src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -232,6 +233,18 @@ uct_dc_mlx5_ep_from_dci(uct_dc_mlx5_iface_t *iface, uint8_t dci)
     return iface->tx.dcis[dci].ep;
 }
 
+static UCS_F_ALWAYS_INLINE void
+uct_dc_mlx5_ep_clear_fc_grant_flag(uct_dc_mlx5_ep_t *ep)
```

Comment:
imp: since this is `inline` function and all callers have `iface`, I think, `iface` argument can be passed from the caller.

or this function isn't called from the fast path?